app/Http/Controllers/Admin/CategoriesController.php

```diff
@@ -130,4 +130,18 @@ public function destroy(Category $category)
         }
         return redirect()->route('category.index');
     }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param Category $category category object
+     *
+     * @return \Illuminate\Http\Response
+    */
+    public function show(Category $category)
+    {
+        $children = $category->children()->with('children')->get();
```

Comment:
->children()->with('children') chỗ này là sao em?

Reply:
à em lấy tất cả những t con của category
cái ->with('children') là em lấy thêm t con cua chính nó ak anh bên model là z nè 
public function children()
    {
        return $this->hasMany(self::class, 'parent_id', 'id');
    }